pytest_watcher/watcher.py

```diff
@@ -74,18 +88,33 @@ def _parse_arguments(args: Sequence[str]) -> Tuple[Path, bool, float, Sequence[s
         default=0.5,
         help="Watcher delay in seconds (default 0.5)",
     )
+    parser.add_argument(
+        "--entrypoint",
```

Comment:
What do you think of `runner` as a more concise name for the argument?
In that case we can rename `pytest_args` to `runner_args`